update setup instructions for R packaging (#97)
esciencecenter-digital-skills/workshop-metadata: ds-rpackaging/schedule-afternoon.md

This workshop is scheduled on four separate days. This allows you to work on
your package in between the sessions, implementing the lessons you learned

> during our meetings together. Please schedule dedicated work time in your

Reviewer: ```suggestion
during the teaching moments. Please schedule dedicated time for homework in your
```

    </table>
  </div>
  <div class="col-md-6">
    <h3>Week 3</h3>
    <table class="table table-striped">
      <tr> <td>13:00</td> <td>Welcome and icebreaker</td> </tr>
      <tr> <td>13:15</td> <td>Recap homework & Lessons learned</td> </tr>
      <tr> <td>14:15</td> <td>Coffee break</td> </tr>
      <tr> <td>14:30</td> <td>Dependencies, Documenting your package</td> </tr>
      <tr> <td>15:30</td> <td>Coffee break</td> </tr>
      <tr> <td>15:45</td> <td>Documenting your package, Vignettes</td> </tr>
      <tr> <td>16:45</td> <td>Wrap-up</td> </tr>
      <tr> <td>17:00</td> <td>END</td> </tr>
    </table>
  </div>
  <div class="col-md-6">
    <h3>Week 4</h3>
    <table class="table table-striped">
      <tr> <td>13:00</td> <td>Welcome and icebreaker</td> </tr>
      <tr> <td>13:15</td> <td>Recap homework & Lessons learned</td> </tr>
      <tr> <td>14:15</td> <td>Coffee break</td> </tr>
      <tr> <td>14:30</td> <td>Vignettes</td> </tr>
      <tr> <td>15:30</td> <td>Coffee break</td> </tr>
      <tr> <td>15:45</td> <td>Sharing our packages</td> </tr>
      <tr> <td>16:45</td> <td>Wrap-up</td> </tr>
      <tr> <td>17:00</td> <td>END</td> </tr>
    </table>
  </div>
</div>

> <p><b>All times in the schedule are in the CET timezone (Amsterdam time).</b></p>

Reviewer: ```suggestion
<p><b>All times in the schedule are in the CE(S)T timezone (Amsterdam time).</b></p>
```

Reviewer: Just a little nitpicking to make this forever applicable :)
